src/Components/AddressForm.tsx

```diff
@@ -167,15 +193,46 @@ export const AddressForm: React.FC<AddressFormProps> = ({
         )}
       </Column>
       <Column span={12}>
-        <Input
+        {/* <Input
+          tabIndex={tabIndex}
+          id="AddressForm_addressLine1"
+          placeholder="Street address"
+          title="Address line 1"
+          value={value?.addressLine1}
+          onChange={changeEventHandler("addressLine1")}
+          error={getError("addressLine1")}
+          data-test="AddressForm_addressLine1"
+        /> */}
+        {/* TODO: Make this work with autocomplete input */}
+        <AutocompleteInput<AddressSuggestion>
```

Comment:
Should probably render this as a fallback if autocomplete is not enabled - otherwise the input has a 🔍 in it. Alternately we could make the AutocompleteInput icon overridable via props.